### PR TITLE
feat: Introduce new properties for the Avatar component [HOMER-2119]

### PR DESCRIPTION
**Pull request by miguelcrespo**
# Purpose of PR

This PR adds the following new properties to the `Avatar` component:
- showColorBorders
- isPrimary
- isActive
- icon

And replace enums for sizes and variants with simple string types

image

## PR Checklist

- [X] I have read the relevant `readme.md` file(s)
- [X] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [X] Tests are added/updated/not required
- [X] Tests are passing
- [X] Storybook stories are added/updated/not required
- [X] Usage notes are added/updated/not required
- [X] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [X] Doesn't contain any sensitive information

### CODE REVIEW

**Review by Lelith — APPROVED** [review]
Looks good to me!

File: packages/components/avatar/README.mdx [hunk 1]
```diff
@@ -22,8 +22,8 @@ import { Avatar, Size, Variant } from '@contentful/f36-avatar';
 
 The Avatar can be rendered in different variants:
 
-- **User** - default variant
-- **App** - default variant
+- **user** - default variant
```
Comment (denkristoffer) [inline packages/components/avatar/README.mdx:25]: Let's keep the lists as they were for the docs website, it looks better and I don't think we need the literal values here, they are listed in both examples and the prop table.

File: packages/components/avatar/src/Avatar.tsx [hunk 2]
```diff
@@ -4,35 +4,35 @@ import { type CommonProps } from '@contentful/f36-core';
...
-}
+export type Variant = 'app' | 'user';
 
 export interface AvatarProps extends CommonProps {
   alt?: ImageProps['alt'];
   isLoading?: boolean;
   size?: Size;
   src?: ImageProps['src'];
   variant?: Variant;
+  showColorBorders?: boolean;
```
Comment (denkristoffer) [inline packages/components/avatar/src/Avatar.tsx:17]:
```suggestion
  showColorBorder?: boolean;
```

I think singular `showColorBorder` fits better here, it's also what you've used yourself in a few places ;D

**Review by denkristoffer — CHANGES_REQUESTED** [review]
Overall this looks good.

- The loading skeleton seems to have broken: https://forma-36-git-feat-homer-2119.colorfuldemo.com/components/avatar#loading-skeleton
- I don't see any differences between active an inactive: https://forma-36-git-feat-homer-2119.colorfuldemo.com/components/avatar#isactive

**Comment by miguelcrespo** [comment]
@denkristoffer Hi Kristoffer, thanks for the review, I just addressed all the comments

**Review by denkristoffer — APPROVED** [review]
Thanks, looks good to me!

**Pull request merged**